tendrl/commons/flows/create_cluster/__init__.py

```diff
@@ -13,6 +13,12 @@
 LOG = logging.getLogger(__name__)
 
 class CreateCluster(flows.BaseFlow):
+    internal = True
```

Comment:
Please define this flow in definitions, this has to be exposed via API and cannot be internal